mne/gui/tests/test_kit2fiff_gui.py

```diff
@@ -71,23 +81,23 @@ def test_kit2fiff_model():
     model.stim_slope = '+'
     events_bin = mne.find_events(raw_bin, stim_channel='STI 014')
 
-    model.stim_chs = '<'
+    model.stim_coding = '<'
     raw = model.get_raw()
     events = mne.find_events(raw, stim_channel='STI 014')
     assert_array_equal(events, events_bin)
 
     events_rev = events_bin.copy()
     events_rev[:, 2] = 1
-    model.stim_chs = '>'
+    model.stim_coding = '>'
     raw = model.get_raw()
     events = mne.find_events(raw, stim_channel='STI 014')
     assert_array_equal(events, events_rev)
 
-    model.stim_chs = 'man'
-    model.stim_chs_manual = list(range(167, 159, -1))
+    model.stim_coding = 'channel'
+    model.stim_chs = "160:161"
```

Comment:
in UMD they have ~20 channels, writing a range is much cleaner for the user than typing in 20 3-digit integers. Which doctoring are you referring to that I missed?